### PR TITLE
Feat/74-discussions-details

### PR DESCRIPTION
**Pull request by evgenibir**
## To Do

#74 

## Visual

image

### CODE REVIEW

File: packages/epics/src/discussions/components/discussion-detail.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { within } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
```
Comment (plitzenberger) [inline packages/epics/src/discussions/components/discussion-detail.stories.tsx:3]: unused imports.

Comment (plitzenberger) [reply packages/epics/src/discussions/components/discussion-detail.stories.tsx:3]:
```suggestion
```

File: packages/epics/src/discussions/components/discussion-detail.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import { DiscussionHead, DiscussionHeadProps } from './discussion-head';
+import { Button, Skeleton } from '@hypha-platform/ui';
+import { RxCross1 } from 'react-icons/rx';
+import { CreatorType } from './discussion-head';
```
Comment (plitzenberger) [inline packages/epics/src/discussions/components/discussion-detail.tsx:4]:
```suggestion
```

File: packages/epics/src/discussions/components/discussion-head.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { Text } from '@radix-ui/themes';
+import { Skeleton, Image } from '@hypha-platform/ui';
```
Comment (plitzenberger) [inline packages/epics/src/discussions/components/discussion-head.tsx:2]: I'm wondering about this `Text` from `@radix-ui/themes`. Can we get rid of it? What is it's benefit and can't we build it ourself?

**Review by plitzenberger — APPROVED** [review]
Beautiful 🤩

File: packages/epics/src/discussions/components/discussion-head.tsx (hunk 3 above)
Comment (evgenibir) [reply packages/epics/src/discussions/components/discussion-head.tsx:2]: > I'm wondering about this `Text` from `@radix-ui/themes`. Can we get rid of it? What is it's benefit and can't we build it ourself?

The component was used more for its meaning than for any specific features, I think we can just get rid of it using new typograhpy classes like text-1, text-2 and other

**Pull request merged**